src/components/pages/Home/RenderHomePage.js

```diff
@@ -2,8 +2,18 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { Button } from '../../common';
 
+// Example for rendering state to app
+import { useSelector, useDispatch } from 'react-redux';
+import { increment, decrement } from '../../../state/actions';
+
 function RenderHomePage(props) {
   const { userInfo, authService } = props;
+
+  // example for rendering state
+  const counter = useSelector(state => state.counter);
+  const isLogged = useSelector(state => state.isLogged);
```

Comment:
I know this is just here for testing and such, but I noticed that there doesn't seem to be any kind of action related to signing in despite there being a reducer for login. 

The best place to probably implement such a thing is wherever they are handling login state currently (can't remember off the top of my head), but this is something that can be done in another PR.